include/api.php

```diff
@@ -2901,21 +2900,34 @@ function api_lists_list($type)
 api_register_func('api/lists/list', 'api_lists_list', true);
 
 /**
- * https://dev.twitter.com/docs/api/1/get/statuses/friends
- * This function is deprecated by Twitter
- * returns: json, xml
+ * @brief Returns either the friends of the follower list
+ *
+ * Note: Considers friends and followers lists to be private and won't return
+ * anything if any user_id parameter is passed.
+ *
+ * @param string $qtype Either "friends" or "followers"
+ * @return boolean|array
+ * @throws ForbiddenException
  */
-function api_statuses_f($type, $qtype)
+function api_statuses_f($qtype)
 {
 	$a = get_app();
 
 	if (api_user() === false) {
 		throw new ForbiddenException();
 	}
 
+	// pagination
+	$count = x($_GET, 'count') ? $_GET['count'] : 20;
```

Comment:
Is it defined in the documentation (Twitter), that "count" is assumed with 20 when not defined?

Reply:
It's what's used elsewhere in our API. Accordingly, it's apparently what Twidere uses explicitly.